Source/TailwindTraders.Mobile/TailwindTraders.Mobile/Features/Home/HomePage.xaml

```diff
@@ -1,222 +1,226 @@
-
-            <carousel:CarouselViewControl
-              HeightRequest="232"
-              InterPageSpacing="{StaticResource NegativeDefaultMarginInt}"
-              ItemTemplate="{StaticResource RecommendedItemTemplate}"
-              ItemsSource="{Binding RecommendedProducts}" />
-
-            <Label
-              Style="{StaticResource HeaderLabelStyle}"
-              Text="POPULAR PRODUCTS" />
-
-            <controls:FixedRatioCarouselViewControl
-              InterPageSpacing="{StaticResource NegativeDefaultMarginInt}"
-              ItemTemplate="{StaticResource PopularItemTemplate}"
-              ItemsSource="{Binding PopularProducts}" />
-
-            <Label
-              Style="{StaticResource HeaderLabelStyle}"
-              Text="PREVIOUSLY SEEN">
-
-              <Label.Margin>
-                <Thickness
-                  Left="{StaticResource 2xDefaultMargin}"
-                  Top="{StaticResource DefaultMargin}"
-                  Right="{StaticResource 2xDefaultMargin}"
-                  Bottom="{StaticResource 2xDefaultMargin}" />
-              </Label.Margin>
-
-            </Label>
-
-            <xfbindablestacklayout:BindableStackLayout
-              ItemTemplate="{StaticResource PreviouslySeenItemTemplate}"
-              ItemsSource="{Binding PreviouslySeenProducts}"
-              Spacing="{StaticResource DefaultMargin}">
-              <View.Margin>
-                <Thickness Bottom="{StaticResource 4xDefaultMargin}" />
-              </View.Margin>
-            </xfbindablestacklayout:BindableStackLayout>
-
-            <VisualStateManager.VisualStateGroups>
-              <VisualStateGroup Name="States">
-
-                <VisualState Name="EverythingOK">
-                  <VisualState.Setters>
-                    <Setter Property="IsVisible" Value="true" />
-                  </VisualState.Setters>
-                </VisualState>
-
-                <VisualState Name="Error">
-                  <VisualState.Setters>
-                    <Setter Property="IsVisible" Value="false" />
-                  </VisualState.Setters>
-                </VisualState>
-
-              </VisualStateGroup>
-            </VisualStateManager.VisualStateGroups>
-          </StackLayout>
-
-        </StackLayout>
-
-      </ScrollView>
-
-      <local:FreeShippingBanner
-        AbsoluteLayout.LayoutFlags="PositionProportional,WidthProportional"
-        AbsoluteLayout.LayoutBounds="0,0,1,AutoSize">
-
-        <local:FreeShippingBanner.Margin>
-          <Thickness
-            Left="{StaticResource 0.5xDefaultMargin}"
-            Top="{StaticResource 0.5xDefaultMargin}"
-            Right="{StaticResource 0.5xDefaultMargin}" />
-        </local:FreeShippingBanner.Margin>
-
-      </local:FreeShippingBanner>
-
-    </AbsoluteLayout>
-
-  </ContentPage.Content>
+<framework:BaseStateAwareContentPage x:Name="homePage"
+                                     x:Class="TailwindTraders.Mobile.Features.Home.HomePage"

```

Comment:
Hey @davidortinau, has Xaml Styler allowed this sort of indents? I'm a little bit out of scope here since have passed some time from my last commits but if you tell me Xaml Styler doesn't keep playing a role here I simply go ahead with the changes :-)

Reply:
I'm running an internal build of VSMac with XAML behavior updates, so that's likely what happened here. I'll have to look at Xaml Styler later to see what happened.